app/scripts/views/mixins/signin-mixin.js

```diff
@@ -103,6 +115,12 @@ define(function (require, exports, module) {
             return this.navigate('signin_bounced', { email: account.get('email') });
           }
 
+          if (AuthErrors.is(err, 'TOTP_REQUIRED') || OAuthErrors.is(err, 'MISMATCH_ACR_VALUES')) {
+            err.forceMessage = t('This request requires two step authentication enabled on your account. ' +
+              '<a target="_blank" href=\'' + TOTP_SUPPORT_URL + '\'>More Information</a>');
```

Comment:
Extracting all of the attributes into a variable is easier for the l10n team to translate:

`<a %(supportLinkAttrs)>More Information</a>`